Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -233,11 +233,26 @@ namespace AzToolsFramework
             for (auto attributeIt = node.MemberBegin(); attributeIt != node.MemberEnd(); ++attributeIt)
             {
                 const AZ::Name& name = attributeIt->first;
+
                 if (name == PropertyEditor::Type.GetName() || name == PropertyEditor::Value.GetName() ||
                     name == PropertyEditor::ValueType.GetName())
                 {
                     continue;
                 }
+                else if (name == PropertyEditor::ParentValue.GetName())
+                {
+                    auto parentValue = PropertyEditor::ParentValue.ExtractFromDomNode(node);
+                    if (parentValue.has_value())
+                    {
+                        auto parentValuePtr = AZ::Dom::Utils::ValueToType<void*>(parentValue.value()).value_or(nullptr);
+                        m_proxyParentNode.m_instances.push_back(parentValuePtr);
```

Comment:
We don't need to check for `nullptr` here before `push_back` because `parentValue.has_value()` guarantees the `parentValuePtr` is not nullptr?

Reply:
I think the code that sets the ptr in `LegacyReflectionBridge` guarantees that it should not be 'nullptr'. But I will add an `AZ_Assert` here to check nullity.